src/hotspot/share/utilities/globalDefinitions_visCPP.hpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 1997, 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 1997, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
You updated the copyright without changing anything in this file.

Reply:
Thanks, I guess I reverted the changes without removing the copyright change.